### ISSUE TITLE
lightdm + x11 crash

### ISSUE THREAD

**Issue by sah4ez** [AsahiLinux/linux#142]
Aflter login got a crashes in RTKit

Versions:
```
lightdm-gtk-greeter --version
** Message: 15:06:50.366: Starting lightdm-gtk-greeter 2.0.8 (Aug 15 2021, 23:37:32)
** Message: 15:06:50.366: [Configuration] Reading file: /usr/share/lightdm/lightdm-gtk-greeter.conf.d/01_debian.conf
** Message: 15:06:50.366: [Configuration] Reading file: /etc/lightdm/lightdm-gtk-greeter.conf

$ uname -a
Linux pc007 6.3.0-asahi-00529-gdad9ac5f8b80 #14 SMP Thu May  4 18:17:30 CEST 2023 aarch64 GNU/Linux

lightdm --version
lightdm 1.26.0
```

journalct output:
```
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: syslog message: UPPipeDCP_H13P.cpp:7476: set_run_mode_safe: no need to defer: 1 -> 0
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: co-processor has crashed
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Version: RTKit: RTKit-1827.120.2.debug - Client: local-ipad14dcp.release
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Crash time: 1544945895
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Exception dump:
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   == Exception taken from EL1h ==
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   PSR    = 0x600012c5
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   PC     = 0x17eaf0
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   ESR    = 0xbe000000
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   FAR    = 0x80000052c
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   SP     = 0x804330
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: 
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x00-x03 = 0000000080010001 000000000000000a 0000000000000000 0000000000000010
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x04-x07 = ffffffff40faa278 0000000000000000 0000000000300000 0000000000055f70
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x08-x11 = 0000000000000001 0000000000000001 000000000000d45c 0000000000000058
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x12-x15 = 0000000000839140 ffffffff40f938c8 0000000044333030 0000000000520ce4
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x16-x19 = ffffffff40fa9b80 0000000000189400 0000000000000000 ffffffff40fa9f50
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x20-x23 = 000000000000000a 000000000000000a 0000000000000000 0000000000857000
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x24-x27 = 000000009fff0000 0000000000000000 0000000000000001 ffffffff40feb8a9
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp:   x28-x31 = ffffffff40febe54 0000000000804360 000000000000d514
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: 
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Unknown crashlog section: 43637374
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Unknown crashlog section: 4372746b
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Unknown crashlog section: 43617343
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Mailbox history (type = 0, index = 0)
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #000@1a6227ec: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #001@1a622b05: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #002@1a622f74: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #003@1a623240: 0000000000000037 0000001000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #004@1a6234c5: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #005@1a63ecbc: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #006@1a63ef39: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #007@1b967ad1: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #008@1b967dc8: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #009@1b967fbd: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #010@1b9682d0: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #011@1b9685e4: 0000000000000037 0000001400000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #012@1b96870d: 0000000000000037 0000000000000342


May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #013@1b9689a4: 0000000000000037 0000001000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #014@1b968a4d: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #015@1b9b19fc: 0000000000000037 0000005000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #016@1b9b1acd: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #017@1b9b33b0: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #018@1b9b3499: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #019@1ba4bdd4: 0000000000000037 0000001000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #020@1ba4bef1: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #021@1d395c75: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #022@1d39601c: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #023@1d39621d: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #024@1d396650: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #025@1d396910: 0000000000000037 0000001000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #026@1d3969e1: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #027@1d3a3ccc: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #028@1d3a3de5: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #029@1d3a3f01: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #030@1d3a46a8: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #031@1d3a477d: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #032@1d3a4adc: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #033@1d405838: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #034@1d40591d: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #035@1d405a75: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #036@1d406224: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #037@1d4062c9: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #038@1d406650: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #039@1d4671ec: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #040@1d46728d: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #041@1d4722b9: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #042@1d472534: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #043@1d4725dd: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #044@1d472a30: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #045@1d4c8d58: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #046@1d4c8e31: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #047@1d4c8fa9: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #048@1d4c96ac: 0000000000000037 0000000000000242


May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #049@1d4c979d: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #050@1d4c9af8: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #051@1d52a698: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #052@1d52a781: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #053@1d58ebd5: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #054@1d58ee9c: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #055@1d58ef49: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #056@1d58f3a4: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #057@1d5edca4: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #058@1d5eddb5: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #059@1d6162a5: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #060@1d616534: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #061@1d61660d: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #062@1d616a5c: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #063@1d64f644: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #064@1d64f749: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #065@1d665065: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #066@1d6652c8: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #067@1d6654ad: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #068@1d6658cc: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #069@1d6b10e4: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #070@1d6b11c5: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #071@1ed0c301: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #072@1ed0c628: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #073@1ed0c711: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #074@1ed0cb6c: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #075@1ed32c30: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #076@1ed32d51: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #077@5bfe69f1: 0000000000000037 0000003c00000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #078@5bfe6d44: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #079@5bfe6f41: 0000000000000037 00000b7800000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #080@5bfe7298: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #081@5bfe7411: 0000000000000037 0000001400000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #082@5bfe7648: 0000000000000037 0000001400000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #083@5bfe771d: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #084@5bfe78f8: 0000000000000037 0000001000000302


May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #085@5bfe79a9: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #086@5c00a724: 0000000000000037 0000005000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #087@5c00a90d: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #088@5c00c194: 0000000000000037 000006ec00000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #089@5c00c271: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #090@5c00c924: 0000000000000037 0000001000000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #091@5c00c9f5: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #092@5c00e508: 0000000000000037 0000001000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #093@5c00e5d5: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #094@5c06da88: 0000000000000037 0000002000000302
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #095@5c06db45: 0000000000000037 0000000000000342
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #096@5c06e1c8: 0000000000000037 0000000c00000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #097@5c06e291: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #098@5c06e38c: 0000000000000037 0000000c00000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #099@5c06e441: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #100@5c06e554: 0000000000000037 0000001000000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #101@5c06e601: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #102@5c06e714: 0000000000000037 0000001000000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #103@5c06e7c5: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #104@5c06e8b0: 0000000000000037 0000000000000242
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #105@5c06e945: 0000000000000037 0000002000000202
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #106@5c06ea7c: 0000000000000037 0000001400000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #107@5c06eb29: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #108@5c06ec9c: 0000000000000037 0000001000000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #109@5c06ed55: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #110@5c0ae3dc: 0000000000000002 0050000000000014
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #111@5c0ae4c0: 0000000000000002 0050000000000015
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #112@5c0ae50d: 0000000000000002 0050000000000014
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #113@5c0ae5d5: 0000000000000002 0050000000000015
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #114@5c0ae768: 0000000000000002 0050000000000016
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #115@5c0ae835: 0000000000000002 0050000000000016
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #116@5c0e953c: 0000000000000002 0050000000000017
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #117@5c0e961d: 0000000000000002 0050000000000017
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #118@5c0e9798: 0000000000000037 0000001400000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #119@5c0e9835: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #120@5c0e9d2c: 0000000000000002 0050000000000018


May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #121@5c0e9df9: 0000000000000002 0050000000000018
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #122@5c0e9ee8: 0000000000000037 0000000c00000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #123@5c0e9f81: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #124@5c0ea1fc: 0000000000000037 0000001400000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #125@5c0ea2a9: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #126@5c0ea48c: 0000000000000037 0000001400000002
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #127@5c0ea521: 0000000000000037 0000000000000042
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Unknown crashlog section: 43636470
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Unknown crashlog section: 43737461
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Mailbox history (type = 4, index = 1)
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #000@05270aac: 0000000000000020 6750313204040020
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #001@052720e1: 0000000000000020 0000000100040000
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #002@05407f88: 0000000000000020 6750306100040011
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #003@05409c19: 0000000000000020 0000000000000000
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #004@1907e8d4: 0000000000000020 4443414c00400010
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #005@190835d9: 0000000000000020 00000000004000d8
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #006@5c08e928: 0000000000000020 6750313200040011
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit:  #007@5c09048d: 0000000000000020 0000000000000000
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: RTKit: Unknown crashlog section: 43737461
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: DCP has crashed
May 05 14:29:36 pc007 kernel: apple-dcp 231c00000.dcp: dcp_hotplug: connected: 0
May 05 14:29:37 pc007 kernel: apple-dcp 231c00000.dcp: setPowerState(0) timeout 1000 ms
May 05 14:29:37 pc007 kernel: apple-dcp 231c00000.dcp: dcp_poweroff() done
May 05 14:29:37 pc007 kernel: ------------[ cut here ]------------
May 05 14:29:37 pc007 kernel: atomic remove_fb failed with -22
May 05 14:29:37 pc007 kernel: WARNING: CPU: 1 PID: 71 at drivers/gpu/drm/drm_framebuffer.c:1130 drm_framebuffer_remove+0x334/0x4a0
May 05 14:29:37 pc007 kernel: Modules linked in: tun rfcomm xt_conntrack nft_chain_nat xt_MASQUERADE nf_nat nf_conntrack_netlink nf_conntrack nf_defrag_ipv6 nf_defrag_ipv4 xfrm_user xfrm_algo xt_addrtype nft_compat nf_tables libcrc32c nfnetlink br_netfilter bridge stp llc qrtr brcmfmac_wcc overlay algif_hash algif_skcipher af_alg bnep binfmt_misc nls_ascii nls_cp437 vfat fat aes_ce_blk brcmfmac aes_ce_cipher brcmutil crct10dif_ce ghash_ce hci_bcm4377 gf128mul bluetooth cfg80211 sha3_ce ecdh_generic sha3_generic rfkill sha512_ce usbcore sha512_arm64 ecc sha2_ce sha256_arm64 snd_soc_macaudio sha1_ce snd_soc_cs42l84 snd_soc_tas2764 snd_soc_apple_mca ofpart snd_soc_core snd_pcm_dmaengine spi_nor snd_pcm mtd tps6598x snd_timer snd soundcore pwm_apple gpio_backlight leds_pwm joydev evdev squashfs pkcs8_key_parser fuse dm_mod loop efi_pstore dax configfs efivarfs ip_tables x_tables autofs4 ext4 crc32c_generic mbcache jbd2 hid_generic dockchannel_hid dwc3 phy_apple_atc udc_core typec roles ulpi usb_common apple_dockchannel
May 05 14:29:37 pc007 kernel: CPU: 1 PID: 71 Comm: kworker/1:1 Tainted: G S                 6.3.0-asahi-00529-gdad9ac5f8b80 #14
May 05 14:29:37 pc007 kernel: Hardware name: Apple MacBook Air (13-inch, M2, 2022) (DT)
May 05 14:29:37 pc007 kernel: Workqueue: events drm_mode_rmfb_work_fn
May 05 14:29:37 pc007 kernel: pstate: 61400009 (nZCv daif +PAN -UAO -TCO +DIT -SSBS BTYPE=--)


May 05 14:29:37 pc007 kernel: pc : drm_framebuffer_remove+0x334/0x4a0
May 05 14:29:37 pc007 kernel: lr : drm_framebuffer_remove+0x334/0x4a0
May 05 14:29:37 pc007 kernel: sp : ffff800008983cc0
May 05 14:29:37 pc007 kernel: x29: ffff800008983d10 x28: ffff45e4d26b1ac8 x27: 0000000000000001
May 05 14:29:37 pc007 kernel: x26: 0000000000000001 x25: 0000000000000028 x24: 00000000ffffffea
May 05 14:29:37 pc007 kernel: x23: ffff45e8303f4080 x22: 0000000000000001 x21: ffffc571897d3ec2
May 05 14:29:37 pc007 kernel: x20: ffff45e4d26b1800 x19: ffff45e4daafd400 x18: 0000000000000011
May 05 14:29:37 pc007 kernel: x17: ffffc57189b89128 x16: ffffffffffffffff x15: 0000000000000004
May 05 14:29:37 pc007 kernel: x14: 0000000000000fff x13: ffffc57189bc5908 x12: 0000000000000003
May 05 14:29:37 pc007 kernel: x11: 0000000000000003 x10: c0000000ffffefff x9 : 2dc7c593f0f6e700
May 05 14:29:37 pc007 kernel: x8 : 2dc7c593f0f6e700 x7 : 62665f65766f6d65 x6 : 722063696d6f7461
May 05 14:29:37 pc007 kernel: x5 : ffffc57189e75fb0 x4 : ffffc57189e66280 x3 : 0000000000000000
May 05 14:29:37 pc007 kernel: x2 : 0000000000000000 x1 : ffff800008983a70 x0 : 0000000000000020
May 05 14:29:37 pc007 kernel: Call trace:
May 05 14:29:37 pc007 kernel:  drm_framebuffer_remove+0x334/0x4a0
May 05 14:29:37 pc007 kernel:  drm_mode_rmfb_work_fn+0x4c/0xc0
May 05 14:29:37 pc007 kernel:  process_one_work+0x1e0/0x410
May 05 14:29:37 pc007 kernel:  worker_thread+0x2dc/0x4f8
May 05 14:29:37 pc007 kernel:  kthread+0xd4/0x170
May 05 14:29:37 pc007 kernel:  ret_from_fork+0x10/0x20
May 05 14:29:37 pc007 kernel: ---[ end trace 0000000000000000 ]---
```

**Comment by jannau** [comment]
known issue on asahi-wip on m2 machines, fixed by #141

**Comment by sah4ez** [comment]
fixed with 
6.3.0-asahi-00538-g3965d153b13f
